errors_collector_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright 2020 YANDEX LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package hasql
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Need test for thread-safety.

Reply:
Are you about include `errorsCollector.Err`  to concurrent test?

I tried to create a separate test, but abandoned it. I really don't understand which statement can be asserted.
So I added [a small hack to the original test](https://github.com/yandex/go-hasql/pull/14/commits/9688671520cc7b134266a2990ebd6c6e0ad06ccd#diff-e9c6f06114388c75d98f0d1d0c0189b92f1ee83affa8829af1a964009184fccfR51).